podcasts/Bookmarks/List/BookmarkPodcastListViewModel.swift

```diff
@@ -0,0 +1,46 @@
+import Combine
+import PocketCastsDataModel
+
+class BookmarkPodcastListViewModel: BookmarkListViewModel {
+    weak var podcast: Podcast?
```

Comment:
Why this needs to be `weak`?

Reply:
hmm. I don't think I had a reason. Fixed here: 2e65ef51894587d9eaee6f68900e4d1cc904c3b5